qmt/tasks/geometry.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Geometry task classes for 1D, 2D and 3D."""
+
+from shapely.geometry import Polygon, LineString
+
+from qmt.data import Geo2DData, Geo3DData, serialised_file
+
+def geometry_2d(parts,edges,lunit='nm',build_order=None):
```

Comment:
Minor suggestion: Since the tasks are now plain functions, a more descriptive naming according to the action they're performing would be nice, like `geometry_2d` -> `build_geometry_2d`.